tests/console/yast2_kdump_enable_kdump.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Enable kdump and accept kdump options.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam suse de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+
+use cfg_files_utils qw(validate_cfg_file);
+use scheduler qw(get_test_suite_data);
+use testapi qw(save_screenshot select_console);
+use utils qw(systemctl zypper_call);
+use YaST::Module;
+
+use YaST::Kdump::StartUpPage;
+use YaST::Kdump::NavigationPage;
+
+sub run {
+    my $startup = YaST::Kdump::StartUpPage->new();
+    my $navigation = YaST::Kdump::NavigationPage->new();
+
+    select_console('root-console');
+    zypper_call('in kdump');
+
+    YaST::Module::open(module => 'kdump', ui => 'ncurses', timeout => 120);
+
+    $startup->enable_kdump();
+    save_screenshot;
+    $navigation->ok()->confirm_reboot_needed();
```

Comment:
we should do this in two steps, sorry if I confused you, the reason is to try to see the screen with the screenshot and make visible the popup, so:
```
my $confirm = $navigation->ok();
save_screenshot;
$confirm->confirm_reboot_needed();
```
make sense? for example in s390x seems that is not visible the popup, we need to be explicit to avoid depending on the machine speed.